crates/turborepo-filewatch/src/package_watcher.rs

```diff
@@ -207,8 +126,8 @@ struct Subscriber {
 /// and some maybe-empty set of workspaces.
 #[derive(Debug)]
 enum State {
-    NoPackageManager,
-    InvalidGlobs(PackageManager),
+    NoPackageManager(Box<dyn std::error::Error + Send + Sync>),
+    InvalidGlobs(Box<dyn std::error::Error + Send + Sync>),
```

Comment:
Minor, but since we're only using the error for the error message via `e.to_string()` we could do that conversion on construction and simplify the type a bit.

Reply:
I'll take a look at making the change. I had originally intended to send the whole `Error` instance back, but due to the watch channel, that turns into a requirement for `Clone`, which in turn propagates pretty far out to all of the different error types.